source/manual/request-fastly-tls-certificate.html.md

```diff
@@ -0,0 +1,55 @@
+---
+owner_slack: "#govuk-platform-health"
+title: Request Fastly TLS certificate
+section: Transition
+layout: manual_layout
+parent: "/manual.html"
+related_applications: [bouncer, transition]
+---
+
+Use the 2nd line account to login to Fastly.
```

Comment:
All of these steps could be prefixed with `1. `, to turn this into a list, which seems a more appropriate format.